stdlib/public/core/UnicodeBreakProperty.swift

```diff
@@ -28,6 +28,7 @@ extension Unicode {
     case v
     case zwj
 
+    @inline(__always)
```

Comment:
The purpose of making sure this gets inlined is to allow the compiler enough visibility to combine the switch statement here with the switch statement in the caller, rather than going switch -> flag -> switch